### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yanngodeau**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,13 +6,13 @@
 
     <groupId>com.orange.fiware</groupId>
     <artifactId>openlpwa-iotagent</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.0.3.1-SNAPSHOT</version>
```
Comment (chazemar) [inline pom.xml]: 1.3.0-SNAPSHOT

File: src/test/java/com/orange/fiware/openlpwa/provider/OpenLpwaMqttProviderTest.java [hunk 2]
```diff
@@ -304,10 +304,10 @@ public boolean matches(Object o) {
                             incomingMessage.getDate() != null &&
                             incomingMessage.getModel() != null &&
                             incomingMessage.getValue() != null &&
-                            incomingMessage.getValue().getPort() == 1 &&
-                            incomingMessage.getValue().getFrameCount() == 8 &&
-                            incomingMessage.getValue().getSignalLevel() == 2 &&
-                            incomingMessage.getValue().getData().equals("ae2109000cf3") &&
+//                            incomingMessage.getoValue().getPort() == 1 &&
```
Comment (chazemar) [inline src/test/java/com/orange/fiware/openlpwa/provider/OpenLpwaMqttProviderTest.java]: remove useless code

**Review by chazemar — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (yanngodeau) [reply pom.xml]: You meant 1.0.3-SNAPSHOT?

Comment (chazemar) [reply pom.xml]: Yes, X.Y.Z is the standard.

**Pull request merged**